css/partials/_member-list.scss

```diff
@@ -1,31 +1,37 @@
+$h3-size: 1.1rem;
+
+%vertical-align {
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
 .member-list {
   display: flex;
   flex-wrap: wrap;
 
-  // The member cards
+  // Member cards
   li {
     @extend .card;
-    flex: 1 0 260px;
+    flex: 1 1 260px;
     margin: 5px;
     background-color: rgba(white, 0.6);
     transition: background-color 400ms ease;
-    border-bottom: solid 5px $theme-primary;
+    box-shadow: 0 5px 0 $theme-primary;
 
     &:hover {
       background-color: white;
```

Comment:
Hover effect nonideal if the card doesn't do anything when clicked

Reply:
I think we can keep the hover, and add a popover to each card where the person can have a bio. Sort of like when you click on a person in discord? Keep the hover, either way - it adds dimension. Example is [here](https://ymath.io/calculus/limits/practice/), I think it looks good?

Reply:
Hmm, still seems to me that something should happen when clicked, but I guess we can add that in another PR